spec/controllers/login_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe LoginController, type: :controller do
+  let(:invite) { create :invite }
+
+  describe 'GET accept_invitation' do
+    let(:expired_invite) { create :expired_invite }
+
+    describe 'with valid token' do
+      it 'accepts invitation' do
+        get :accept_invitation, params: { foodcoop: FoodsoftConfig[:default_scope], token: invite.token }
+        expect(response).to have_http_status(:success)
+        expect(response).to render_template('login/accept_invitation')
+      end
+    end
+
+    describe 'with invalid token' do
+      it 'redirects to login' do
+        get :accept_invitation, params: { foodcoop: FoodsoftConfig[:default_scope], token: invite.token + 'XX' }
+        expect(response).to have_http_status(:redirect)
+        expect(response).to redirect_to(login_url)
+        expect(flash[:alert]).to match(I18n.t('login.controller.error_invite_invalid'))
```

Comment:
what do you think about a helper method for checking the `flash`? it's the same pattern multiple times

Reply:
I'm not sure if I grasp your idea correctly ...
what i'd come up with could be smth like 
```ruby
  def expect_flash_alert_to_match(msg)
    expect(flash[:alert]).to match(msg)
  end
 ```
I wouldn't see so much benefit of this refactoring ... But If you prefer it, I can do that ofc